lldb/test/API/lang/swift/async/unwind/unwind_in_all_instructions/main.swift

```diff
@@ -0,0 +1,49 @@
+func async_work() async {
+  print("async working")
+}
+func work() {
+  print("working")
+}
+
+func ASYNC___1___(cond: Int) async -> Int {
+  work()
+  await async_work()
+  for i in 1...3 {
+    work()
+    await async_work()
+    work()
+    if (cond + i == 3) {
+      await async_work()
+      print("breaking here!")
```

Comment:
fwiw, "breaking here" led me to think that this was used as source pattern regex.

Reply:
Rewording as "exiting loop here"